README.md

```diff
@@ -1,18 +1,18 @@
 <div align="center">
     
-<img src="https://raw.githubusercontent.com/ietf-tools/common/main/assets/logos/wagtail-site.svg" alt="IETF Wagtail Website" height="125" />
+<img src="https://raw.githubusercontent.com/ietf-tools/common/main/assets/logos/www.svg" alt="IETF WWW" height="125" />
```

Comment:
This is not going to work. We need @NGPixel to create a new one. Also "IETF WWW" doesn't sound right to me.

Reply:
As per the slack conversation, lets's keep this as it is.